wdl-lint/src/rules/blank_lines_between_elements.rs

```diff
@@ -44,15 +44,13 @@ fn excess_blank_line(span: Span) -> Diagnostic {
     Diagnostic::note("extra blank line(s) found")
         .with_rule(ID)
         .with_highlight(span)
-        .with_fix("remove the blank line(s)")
```

Comment:
Is the idea simply that we are removing fix messages that are trivially understood from the diagnostic message? If so, is that really something that is worth unwinding? It makes it a bit shorter, but, sometimes, I scan my errors for the "fix" part first and only dive into the actual error when it's needed.

Reply:
After skimming through the changes to the diagnostics below, I really don't like it. Visually, I think the fix line provides a nice closure to the messages. Additionally, even if the fix messages are sometimes redundant, the value of having the same location in each error message to determine how to fix something makes the experience silently _much_ more pleasant IMO.

Reply:
Quoting what I wrote in #193 as it contains my entire argument:
> Many of our `fix` messages are redundant, obvious, or otherwise unhelpful. We should use the `fix` feature more mindfully. That means supplying the "corrected" code or **adding** something to the diagnostic that isn't already conveyed. And if those can't be accomplished, the `fix` should be omitted.

Personally I find these "unhelpful" `fix` messages to be visual overload. It's just more text to read.

Do @peterhuene or @adthrasher have an opinion one way or the other?

Reply:
Also worth noting that not every diagnostic on `main` uses the `fix` feature. Do you (@claymcleod ) want to see a `fix` used _everywhere_? We should pick a convention and stick to it. So if I'm outvoted (which I'd be fine with), I would advocate for adding new `fix` messages to diagnostics that currently don't have one.

Reply:
I think I'm in agreement with @claymcleod. I like having the `fix` as a closing delimiter for the messages. Some of the `fix` messages will need revision, but I don't necessarily think that some redundancy is problematic. The lint message should point out the problem and the `fix` should describe the action necessary to correct it. For spacing, that's likely to be somewhat redundant, but I like the consistency. Plus the user should know what needs to be done from the `fix` rather than trying to determine from the error/warning itself.

Reply:
I think both of yalls points make sense. I can go back and revert all the deletions.

> Also worth noting that not every diagnostic on `main` uses the `fix` feature. Do you want to see a `fix` used _everywhere_? We should pick a convention and stick to it. So if I'm outvoted (which I'd be fine with), I would advocate for adding new `fix` messages to diagnostics that currently don't have one.

I will also add `fix` messages to all our diagnostics that currently don't have one.